sklearn/datasets/_base.py

```diff
@@ -22,6 +23,10 @@
 
 from urllib.request import urlretrieve
 
+DATA_MODULE = "sklearn.datasets.data"
+DESCR_MODULE = "sklearn.datasets.descr"
+IMAGES_MODULE = "sklearn.datasets.images"
```

Comment:
Both string and module work. Using string here so that it can be more conveniently serialized (e.g. `pickle`) as part of a `Bunch`.